presto-spi/src/main/java/io/prestosql/spi/HostAddress.java

```diff
@@ -247,6 +247,12 @@ public InetAddress toInetAddress()
         return InetAddress.getByName(getHostText());
     }
 
+    public InetAddress[] getAllInetAddresses()
```

Comment:
Could you elaborate on what part seems "specific"? Just want to understand it better.

`HostAddress` currently has `toInetAddress` method, which effectively returns `getAllInetAddresses()[0]`, so that one seems even more specific. would you recommend moving that out too?

Reply:
I meant that it is only used in once place, and doesn't seem needed as a shared method. But I didn't realize we already had `toInetAddress` and this method fits with that, so it now seems fine to add it.

We should have it return a `List` rather than an array, per Effective Java.

Reply:
removed the special case.